styles.css

```diff
@@ -427,3 +433,196 @@ h4 {
   position: relative;
   left: 30%;
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    display: contents;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navbar .desktop-menu-bar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    gap: 46px;
+    padding-right: 43px;
+    color: #ff195e;
+  }
+
+  .desktop-menu-bar a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
```

Comment:
- [ ] Please notice there is no text-underline in the original design for the navbar links :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213240628-43f3908e-62b6-418d-9ce8-b8210551acbb.png)

let's fix this by adding a `[text-underline](text-decoration: none;)`